### PR TITLE
ADD email in CODE_OF_CONDUCT.md

### LINKED ISSUE #2325: missing email in CODE_OF_CONDUCT.md
### What would you like to share or ask?

I suggest that in the **Enforcement** section the `[INSERT CONTACT METHOD]` will be replaced with @rupali-codes email address. or an email address.

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by JuanPabloDiaz**
## Fixes Issue

fixes #2325

## Changes proposed

The Code of conduct now has @rupali-codes email: [rupali.codes@gmail.com](http://rupali.codes@gmail.com/)

### CODE REVIEW

**Review by CBID2 — APPROVED** [review]
LGTM. @rupali-codes or @aftabrehan, it's your turn

**Review by aftabrehan — APPROVED** [review]

**Review by rupali-codes — COMMENTED** [review]
can you please replace the email with `linkshub.opensource@gmail.com`

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ representative at an online or offline event.
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
 reported to the community leaders responsible for enforcement at
-[INSERT CONTACT METHOD].
+rupali.codes@gmail.com.
```
Comment (CBID2) [inline CODE_OF_CONDUCT.md:64]: [typo]: this the correct email 
```suggestion
linkshub.opensource@gmail.com.
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @JuanPabloDiaz. It turns out the email you put is incorrect. Please fix and tag me and @rupali-codes for another review

**Review by CBID2 — APPROVED** [review]
Looks good @JuanPabloDiaz! :) In the future, click on the “Commit” button when you see an embedded suggestion in your pull requests.

**Pull request merged**